app/components/captcha_submit_button_component.html.erb

```diff
@@ -0,0 +1,35 @@
+<%= content_tag(
+      :'lg-captcha-submit-button',
+      **tag_options,
+      'recaptcha-site-key': IdentityConfig.store.recaptcha_site_key_v3,
+      'recaptcha-action': action,
+    ) do %>
+  <%= f.error(:recaptcha_token) %>
+  <%= f.input(:recaptcha_token, as: :hidden) %>
+  <% if show_mock_score_field? %>
+    <%= render AlertComponent.new(text_tag: :div, class: 'display-inline-block margin-top-0 margin-bottom-2') do %>
+      <div class="display-flex flex-align-center">
+        <%= f.label(
+              :recaptcha_mock_score,
+              'reCAPTCHA score:',
+              class: 'usa-label float-left margin-y-0 margin-right-1',
+            ) %>
+        <%= f.text_field(
+              :recaptcha_mock_score,
+              value: '1.0',
```

Comment:
do we want to document/indicate what the range on this is? 0-1?

Reply:
Yeah, maybe can somehow link to https://developers.google.com/recaptcha/docs/v3#interpreting_the_score

Reply:
I'd just update the label text to say it? 🤷

Reply:
Added score range to label in 646811cdd.